.github/workflows/browser-test.yml

```diff
@@ -0,0 +1,78 @@
+name: browser test
+
+on: [pull_request, workflow_dispatch]
+jobs:
+  build:
+    name: Build website assets
+    uses: ZTL-ARTCC/Current_Website/.github/workflows/build.yml@main
+    with:
+      artifact-name: 'ztl-deploy-artifact-v1'
```

Comment:
This never gets used, so we should either use this (preferable if we can with it working), or not depend on this